spk/davfs2/src/installer.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+
+# Package
+PACKAGE="davfs2"
+DNAME="davfs2"
+
+DAVUSER="davfs2"
+DAVGROUP="davfs2"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+    # Install busybox stuff
+    ${INSTALL_DIR}/bin/busybox --install ${INSTALL_DIR}/bin
+
+    # Add symlink
+    mkdir -p /usr/local/bin
+    ln -s ${INSTALL_DIR}/sbin/mount.davfs /usr/local/bin/mount.davfs
+    ln -s ${INSTALL_DIR}/sbin/umount.davfs /usr/local/bin/umount.davfs
+    
+    ${INSTALL_DIR}/bin/addgroup ${DAVGROUP}
+    ${INSTALL_DIR}/bin/adduser -h /var/cache/davfs2/ -s /sbin/nologin -G ${DAVGROUP} -D ${DAVUSER}
```

Comment:
- Create a homedir for the user in ${INSTALL_DIR}/var, and use that. Don't use `/var/*`
- Add the user to the `users` group initially, additional groups are explained in my other comment.